### PR TITLE
metrics for txn region count

### PR DESCRIPTION
**Pull request by AndreMouche**
Hi,all, 
      This PR is for [issues_2623](https://github.com/pingcap/tidb/issues/2623). @disksing @coocood PTAL

### CODE REVIEW

File: store/tikv/2pc.go [hunk 1]
```diff
@@ -190,6 +190,18 @@ func (c *twoPhaseCommitter) doActionOnKeys(bo *Backoffer, action twoPhaseCommitA
...
 
+	actionLabel := "2pc"
+	switch action {
+	case actionPrewrite:
+		actionLabel += "_prewrite"
+	case actionCommit:
+		actionLabel += "_commit"
+	case actionCleanup:
+		actionLabel += "_cleanup"
+	}
```
Comment (disksing) [inline store/tikv/2pc.go]: It will be better to define a method for `twoPhaseCommitAction`.

File: store/tikv/2pc.go [hunk 2]
```diff
@@ -50,6 +50,11 @@ func (ca twoPhaseCommitAction) String() string {
 	return "unknown"
 }
 
+// MetricsTag return detail tag for metrics
```
Comment (disksing) [inline store/tikv/2pc.go]: return -> returns, and append a . at the end.

**Comment by disksing** [comment]
The rest LGTM. PTAL @hhkbp2

**Comment by disksing** [comment]
LGTM

**Comment by AndreMouche** [comment]
@hhkbp2 PTAL

**Comment by shenli** [comment]
LGTM

**Comment by AndreMouche** [comment]
@hhkbp2 PTAL

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**